### PR TITLE
Add VScode tasks to build and flash efr32 apps with selected boards.

### PR DESCRIPTION
**Pull request by jmartinez-silabs**
#### Problem
no problem just adding a vscode task

#### Change overview
Add two tasks in .vscode/task.json
Update efr32.py and targets.py to support efr32 board selection

#### Testing
Use the new tasks to build and flash a few examples on efr32 boards

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -170,6 +170,30 @@
                     "${workspaceFolder}/src/test_driver/mbed/unit_tests/build"
                 ]
             }
+        },
+        {
+            "label": "Build specific EFR32 example",
+            "type": "shell",
+            "command": "scripts/run_in_build_env.sh",
+            "args": [
+                "./scripts/build/build_examples.py --target efr32-${input:efr32Board}-${input:efr32App}  build"
```
Comment (andy31415) [inline .vscode/tasks.json:179]: Please  move this to the separate build rule that covers everyting

File: .vscode/tasks.json [hunk 2]
```diff
@@ -292,6 +316,35 @@
...
+                "window-covering"
+            ],
+            "default": "light"
+        },
+        {
+            "type": "pickString",
+            "id": "efr32Board",
+            "description": "What EFR32 board are you building for?",
+            "options": [
+                "brd4161a",
```
Comment (andy31415) [inline .vscode/tasks.json:338]: add these to exampleTarget ? That list is autocomplete so just having  it super large does not seem very painful.

**Review by woody-apple — APPROVED** [review]

File: .vscode/tasks.json (hunk 1 above)
Comment (jmartinez-silabs) [reply .vscode/tasks.json]: Deleted that task. All boards can now be build with the Build specific example task.

File: .vscode/tasks.json (hunk 2 above)
Comment (jmartinez-silabs) [reply .vscode/tasks.json]: Ok, i brought extended exampleTarget to contain all examples and boards and removed the new pickStrings I previously added

**Review by jepenven-silabs — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**